cookbooks/bcpc-hadoop/recipes/os_basic_queries.rb

```diff
@@ -0,0 +1,75 @@
+# Get a list of cluster nodes to monitor
+monitored_nodes_objs = get_all_nodes.map { |n| n if not n.hostname =~ /bootstrap/ }.compact
+
+# Graphite queries which specify property to query and alarming trigger, severity(maps to zabbix's api -> trigger -> priority)
+# and owner who the trigger is routed to for resolution
+# Queries are structured as they appear in graphite dashboard
+# Refer Zabbix API(https://www.zabbix.com/documentation/2.2/manual/api/reference/) for the various attributes used
+# The following list is supposed to be a skeleton which should be extended/edited according to the deployment scenario
+# By default items, triggers actions are disabled and won't be monitored. To enable them, set "enable=true" in the json structure
+monitored_nodes_objs.each do |node_obj|
+  host = node_obj.hostname
+  node.default["bcpc"]["hadoop"]["graphite"]["queries"]["#{host}"] = { 
```

Comment:
Hrm, I'm not sure about storing another hosts' monitoring metadata in this host' node object. This may cause an unalterably large node object. Can we structure this in a way where each node holds its own to-be-monitored attributes even if it is not registering the triggers? (Or perhaps it would get the defaults unless it provides an override?)